Sources/Sentry/SentryClient.m

```diff
@@ -253,6 +255,17 @@ - (SentryEvent *_Nullable)callEventProcessors:(SentryEvent *)event
     return newEvent;
 }
 
+- (void)setUserInfo:(NSDictionary *)userInfo
+        withEvent:(SentryEvent *)event
+{
+    if (nil != event && nil != userInfo && userInfo.count > 0) {
```

Comment:
`l`: I think we don't have to check if the `event` is `nil` here.

Reply:
I wouldn't know unless I looked at all call sites. Since this is free, and you used `l`, I'll keep it